addon/-private/utils.js

```diff
@@ -12,11 +12,11 @@ function modelHasAttributeOrRelationshipNamedType(modelClass) {
   return get(modelClass, 'attributes').has('type') || get(modelClass, 'relationshipsByName').has('type');
 }
 
-/*
+/**
```

Comment:
Do you mind removing the extra `*`s that were added in this file? YUIDoc sees these as API docs even though these functions are never made available to the user.